### PR TITLE
Unified log path for pre-trained models, saved test-time outputs, and added Bland-Altman plots

### PR DESCRIPTION
**Pull request by yahskapar**
This PR creates a unified log path for pre-trained models, saved test-time outputs, and Bland-Altman plots. In the near future, it will also include training and validation loss curves, as well as learning rate curves. 

I recommend not paying too much attention to f98534c08ff38ed3a4d716d2dc695517f50e2b69, as that's a very simple change (enabling Bland-Altman plots by default) applied to all of the config files.

### CODE REVIEW

**Comment by yahskapar** [comment]
Also added fixed training and validation loss, as well as learning rate, plots now in 41324c5.

**Comment by yahskapar** [comment]
Please take a look when you have a chance @girishvn @xliucs, and please ignore commits with many simple file changes like f98534c and 7c0e8a8. Those contain very small, single line changes to set a default in the config.

**Review by xliucs — APPROVED** [review]
LGTM
